dashboard/db/migrate/20170117185757_add_course_workshop_info_to_regional_partner_district_mapping.rb

```diff
@@ -0,0 +1,6 @@
+class AddCourseWorkshopInfoToRegionalPartnerDistrictMapping < ActiveRecord::Migration[5.0]
+  def change
+    add_column :regional_partners_school_districts, :course, :string
+    add_column :regional_partners_school_districts, :workshop_info, :string
```

Comment:
What is `workshop_info`? Will that be just the date string describing the workshop? Please add a `comment:` here.

Should we have a workshop id field?